### PR TITLE
Change region parameter handling

### PR DESCRIPTION
**Pull request by cd-at-iteratec**
chore: make region parameter deprecated and use region value from environmentconfig

### CODE REVIEW

File: charts/v1beta1/tenantfilesystem/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.4.0]
```
Comment (mithie) [inline charts/v1beta1/tenantfilesystem/CHANGELOG.md:8]: can you add the link on the bottom of this file?

File: charts/v1beta1/tenantfilesystem/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
...
+
+### Change
+* CRD parameter `region` got deprecated. Will removed in the next versions.
```
Comment (mithie) [inline charts/v1beta1/tenantfilesystem/CHANGELOG.md:11]: >
```suggestion
* CRD parameter `region` got deprecated. Will be removed in the next major version.
```

**Review by mithie — CHANGES_REQUESTED** [review]
only two very minor issues, rest looks fine

File: charts/v1beta1/tenantfilesystem/CHANGELOG.md (hunk 1 above)
Comment (cd-at-iteratec) [reply charts/v1beta1/tenantfilesystem/CHANGELOG.md:8]: Done

File: charts/v1beta1/tenantfilesystem/CHANGELOG.md (hunk 2 above)
Comment (cd-at-iteratec) [reply charts/v1beta1/tenantfilesystem/CHANGELOG.md]: done

**Review by mithie — APPROVED** [review]

**Pull request merged**